pkgs/development/python-modules/textual/default.nix

```diff
@@ -7,6 +7,7 @@
   platformdirs,
   poetry-core,
   pytest-aiohttp,
+  pytest-asyncio,
```

Comment:
This appears to be unused. Also, I don't think it's necessary to include this to the check deps as `asyncio` already seems available to `pytest`: 

```
plugins: aiohttp-1.0.5, asyncio-0.23.8, xdist-3.6.1, syrupy-4.8.0, time-machine-2.15.0
asyncio: mode=Mode.AUTO
```

Reply:
You are right, forgot to remove. :+1: